app/src/main/java/com/brainwallet/ui/screen/inputwords/InputWordsState.kt

```diff
@@ -0,0 +1,11 @@
+package com.brainwallet.ui.screen.inputwords
+
+data class InputWordsState(
+    val bip39Words: List<String> = emptyList(),
+    val suggestionsSeedWords: List<String> = emptyList(),
+    val seedWords: Map<Int, String> = (0..11).associateWith { "" } //12 seed words
```

Comment:
not that there is a chance that we are migrating to more but I'd like this in a global constant.  `globalSeedWordCount = 12`

Reply:
noted will think on it later